### DISCUSSION TITLE
Asset Manager - how to not store assets locally?

### DISCUSSION THREAD

**Discussion by skru** [GrapesJS/grapesjs#4111] [General]
Implementing a backend for our grapes AssetManager. Having trouble with assets being added to local storage (inbuilt-local or indexeddb plugin). 

    let am = editor.AssetManager
    am.add(["image.png"])

So AssetManager.add() adds to storage too, is there anyway to disable this? (we never want assets in local storage)

Thanks

**Comment by artf** [comment]
Try to monkey patch the store method in Assets module
```js
editor.Assets.store = () => {}
```

**Reply by skru** [reply]
Nice solution! Thanks worked a treat
